infra/vsts_agent_windows.tf

```diff
@@ -14,12 +14,112 @@ locals {
       size       = 6,
       assignment = "default",
       disk_size  = 400,
+      pinning    = "",
     },
     {
       name       = "ci-w2"
-      size       = 0,
-      assignment = "default",
+      size       = 1,
+      assignment = "pinned",
       disk_size  = 400,
+      pinning    = <<EOF
+
+# Set workdir <> job mappings
+New-Item -ItemType Directory -Path 'D:\a'
+New-Item -ItemType Directory -Path 'D:\a\SourceRootMapping'
+New-Item -ItemType Directory -Path 'D:\a\SourceRootMapping\cb88e308-485c-40f9-81b5-dcabba9e55d2'
```

Comment:
there are a lot of magic uuids in here. I recommend adding coments that explain how you got them and how to update them when needed.

Reply:
To be clear, I did not craft those files: this is all taken literally from the output of one machine (that happened to have run all three jobs) on [this step](https://github.com/digital-asset/daml/pull/12597/files).

I _think_ the UUIDs are the IDs of the jobs in Azure, but I wouldn't recommend trying to change any of this. If we ever get any new job in Azure, I'd still rely on that output and take it as is, rather than try to craft these values.